server/reviewer/queries.ts

```diff
@@ -0,0 +1,12 @@
+import { Reviewer } from 'server/models';
+
+export const createReviewer = (inputValues) => {
```

Comment:
In `createReview` I noticed that you destructured `inputValues` near the beginning of the function to make it clear what needs to be provided there...if I had to guess that was because you had to carefully read the function body in order to tell what values were required!

Here and now you have the power to forestall a similar fate for your colleagues by either destructuring in the function signature or (better) declaring a type for `inputValues` :)